ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/name/vch-creation-wizard-name.html

```diff
@@ -1,16 +1,48 @@
-<p class="smaller-p-body-text">
-  This wizard allows you to create a virtual container host by following a few steps.
+<p>
+  This wizard builds a command that creates a virtual container host.
+  You can copy the command from the Summary page into your command line interface.
 </p>
+
 <form [formGroup]="form" novalidate>
   <section class="form-block">
-    <div class="form-group">
-      <label for="nameInput">Name <span class="red-text">*</span></label>
-      <label for="nameInput" aria-haspopup="true" role="tooltip" class="tooltip tooltip-validation tooltip-lg tooltip-top-left"
-        [class.invalid]="form.get('name').invalid
-                    && (form.get('name').dirty || form.get('name').touched)">
-        <input id="nameInput" size="50" type="text" placeholder="virtual-container-host" formControlName="name">
-        <span class="tooltip-content">{{ formErrMessage }}</span>
-      </label>
+
+    <div class="form-group row">
+      <div class="col-xs-2">
+
+        <label for="nameInput" class="required">
+          Name
+        </label>
+
+      </div>
+      <div class="col-xs-6">
+
+        <label for="nameInput"
+               aria-haspopup="true"
+               role="tooltip"
+               class="form-control tooltip tooltip-validation tooltip-lg tooltip-top-left"
+               [class.invalid]="form.get('name').invalid && (form.get('name').dirty || form.get('name').touched)">
+
+          <input id="nameInput"
+                 class="form-control"
+                 size="50"
```

Comment:
This seems rather arbitrary. Using 'size' can be inconsistent across browsers. Maybe we can use a CSS equivalent size such as 'ch'. Add to style sheet if feasible. But either way let's validate against the mockup and check cross browser.